src/Console/ConfigurationResolver.php

```diff
@@ -889,7 +889,7 @@ private function resolveFinder(): iterable
         $isIntersectionPathMode = self::PATH_MODE_INTERSECTION === $this->options['path-mode'];
 
         $paths = array_map(
-            static fn (string $path) => realpath($path),
+            static fn (string $path): string => realpath($path), // @phpstan-ignore return.type
```

Comment:
when strict-types is enabled this will trigger a fatal error
https://3v4l.org/gfvai

I think you should use a real cast, or handle a possible `false` return

```suggestion
            static fn (string $path): string => (string) realpath($path), // @phpstan-ignore return.type
```

Reply:
> when strict-types is enabled this might trigger a fatal error

This is the intention of this change. I don't want to continue the code with `false` (or casted to empty string), it should fail here. (we use strict types)
Because it is very unlikely that `realpath` is failing here, I did not add an own exception and use the `TypeError` coming from PHP instead. 
(The paths are already checked before in `$this->getPath()`, if they are readable)

Reply:
`''` (`(string) false`) would also make no meaningful sense as one of paths